.rubocop.yml

```diff
@@ -382,3 +392,30 @@ Style/SlicingWithRange:
 # Added in v0.84
 Lint/DeprecatedOpenSSLConstant:
   Enabled: true
+
+# the default style recently changed, but it's not worth changing
+Rails/FilePath:
+  EnforcedStyle: arguments
+
+# This one doesn’t always make sense, it’s mostly the swagger,
+# factories and pact provider states — which need a tear_down block regardless
+Lint/EmptyBlock:
+  Enabled: false
+
+Lint/NoReturnInBeginEndBlocks:
+  Exclude:
+    - 'app/services/form1010cg/service.rb'
+
+# Will implement this in another PR
+# LOC changed is too high
+Lint/SymbolConversion:
+  Enabled: false
+
+Style/HashConversion:
```

Comment:
HashConversion, NegatedIfElseCondition and RedundantArgument will be addressed in a separate PR as well to get below the dangerbot LOC allowed